### PR TITLE
Add support for django 1.11

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

**Comment by jlward** [comment]
The template tags are not being tested. And there are at least two problems in that file. We'll need to add some simple smoke test.

File: tagging/models.py [hunk 1]
```diff
@@ -1,22 +1,21 @@
 """
 Models and managers for generic tagging.
 """
-# Python 2.3 compatibility
-try:
-    set
-except NameError:
-    from sets import Set as set
```
Comment (bocribbz) [inline tagging/models.py:8]: `set` wasnt used in this file at all?

Comment (bocribbz) [reply tagging/models.py:8]: or what was this for?

File: tagging/models.py [hunk 2]
```diff
@@ -474,14 +474,17 @@ class Meta:
     def __unicode__(self):
         return self.name
 
+    def __str__(self):
+        return self.__unicode__()
```
Comment (bocribbz) [inline tagging/models.py:478]: why do we need `__str__` too? is this for py3?

Comment (jlward) [reply tagging/models.py:478]: Yes, python3.

File: tagging/models.py (hunk 1 above)
Comment (jlward) [reply tagging/models.py:8]: Old python didn't have `set` in the standard lib. We are not using old python any longer :)

File: tagging/templatetags/tagging_tags.py [hunk 3]
```diff
@@ -14,10 +14,10 @@ def __init__(self, model, context_var, counts):
         self.counts = counts
 
     def render(self, context):
-        model = get_model(*self.model.split('.'))
+        model = apps.get_model(*self.model.split('.'))
         if model is None:
             raise TemplateSyntaxError(_('tags_for_model tag was given an invalid model: %s') % self.model)
-        context[self.context_var] = Tag.objects.usage_for_model(model, counts=self.counts)
+        context[self.context_var] = list(Tag.objects.usage_for_model(model, counts=self.counts))
```
Comment (bocribbz) [inline tagging/templatetags/tagging_tags.py:20]: whats the reason for `list` here?

Comment (jlward) [reply tagging/templatetags/tagging_tags.py:20]: Yea, the __str__ method of a QuerySet is different in django 1.11.

Comment (bocribbz) [reply tagging/templatetags/tagging_tags.py:20]: what does `str` have to do here?

Comment (jlward) [reply tagging/templatetags/tagging_tags.py:20]: We are rendering the result of this as a string in a template. It's the new test added. I really only added it so this file would be imported (showing problems with django 1.11 imports).

Comment (bocribbz) [reply tagging/templatetags/tagging_tags.py:20]: ok, that test was a big foggy too..
creating 1 instance with 1 tag didnt felt like smoke testing.

**Review by bocribbz — APPROVED** [review]
Looks great!

**Pull request merged**